### PR TITLE
Allow go cli to be configurable

### PR DESCRIPTION
**Pull request by rcrichton**
The go cli is now configurable via config.yml to allow different distributions of the cli to be created with a different set of packages and default config options.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:fermium-buster
+FROM node:fermium-buster AS base
 
 WORKDIR /instant
```
Comment (MattyJ007) [inline Dockerfile:3]: Perhaps we could name this something more generic to allow more logical reuse?
`/packages` ?
`/components` ?

File: goinstant/goinstant.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ type Config struct {
 	Image              string    `yaml:"image"`
 	DefaultEnvironment string    `yaml:"defaultEnvironment"`
 	Packages           []Package `yaml:"packages"`
-	DisableKuberneses  bool      `yaml:"disableKubernetes"`
+	DisableKubernetes  bool      `yaml:"disableKubernetes"`
```
Comment (MattyJ007) [inline goinstant/goinstant.go:28]: Good catch 😉

File: Dockerfile (hunk 1 above)
Comment (rcrichton) [reply Dockerfile:3]: Agreed, but that might break packages that expect config to be in the instant volume. We'd have to re-look at how the volume is mounted before we make changes. I'm actually thinking that we need to do some thinking and re-work of the how the instant volume works. I think we need to ensure the latest from the image is always loaded in there. I think we should look at this as a separate piece of work.

File: goinstant/goinstant.go [hunk 3]
```diff
@@ -30,7 +48,10 @@ var customOptions = customOption{
 func stopContainer() {
 	commandSlice := []string{"stop", "instant-openhie"}
 	suppressErrors := []string{"Error response from daemon: No such container: instant-openhie"}
-	runCommand("docker", suppressErrors, commandSlice...)
+	_, err := runCommand("docker", suppressErrors, commandSlice...)
+	if err != nil {
+		gracefulPanic(err, "")
```
Comment (Mark-Labuschagne) [inline goinstant/goinstant.go:53]: This call to gracefulPanic results in an infinite loop on error. I fixed this in a separate PR.

**Review by Mark-Labuschagne — APPROVED** [review]

**Review by MattyJ007 — APPROVED** [review]

**Pull request merged**